### DISCUSSION TITLE
can you offer the training of siammask?

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#249] [General] [answered]
can you offer the training of siammask?
thanks

**Comment by ZhiyuanChen** [comment]
What do you mean by training of SiamMask?

**Comment by ghost** [comment]
pysot里面没有 siammask的训练代码吗？怎么训练siammask呢？



---Original---

**Comment by ghost** [comment]
can you offer the training code of siammask in pysot?

**Comment by ZhiyuanChen** [comment, marked as answer]
> can you offer the training code of siammask in pysot?

https://github.com/STVIR/pysot/blob/master/tools/train.py
